parcels/compiler.py

```diff
@@ -10,7 +10,7 @@ def getuid():
 
 
 def get_package_dir():
-    return path.abspath(path.join(path.dirname(__file__), path.pardir))
+    return path.abspath(path.join(path.dirname(__file__)))
```

Comment:
What exactly does this do? Why is this necessary now? Keen to understand it's workings better

Reply:
- `path.pardir` is whatever indicates "parent directory". In a Unix / Posix world, this is `..`.
- `path.dirname` returns the directory containing `__file__`.
- `__file__` is the full path (incl. potential symlinks etc.) of the file containing this line of code.
- `path.join` joins it inputs to a properly qualified path, taking care of extra `/` (Posix) or `\` (Windows) etc.

My change basically means (in the context of <https://github.com/willirath/parcels/blob/1b74b40438fc108773b8f6c14a1c887218cf9a80/parcels/compiler.py#L66> which is the only time it is used): "Look for `include/` in the same directory as `compiler.py` and not one level above."

And I note that I should drop the `path.join` which does not do anything here.